.github/sync_labels.py

```diff
@@ -0,0 +1,905 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+r"""
+Python script to sync labels that are migrated from Trac selection lists.
+For more information see https://github.com/sagemath/sage/pull/35172
+"""
+
```

Comment:
The same here. It is true that our set of labels came from Trac practices. But what the script does is to synchronize the labels with the GitHub state of PR. The origin of the labels is irrelevant here.

Reply:
Instead of "labels that are migrated from Trac selection lists", you may say "status and priority labels".

Reply:
Done!